resources/frontend/lobby.css

```diff
@@ -411,4 +411,9 @@ input[type="button"]:active,
 #reconnect-dialog {
     /* As this dialog is very important, it should always be on the top. */
     z-index: 100;
-}
\ No newline at end of file
+}
+
+#drawer-name {
```

Comment:
Maybe we should rename this a bit, as it's just the drawers name in a very specific scenario.